src/scenes/Login/Login.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import { MockApp } from '../../App';
+import { LoginForm } from './LoginForm';
+
+export default { title: 'Scenes.Login' };
+
+export const _LoginForm = () => (
```

Comment:
What's the leading underscore for?

Reply:
We're already importing `LoginForm` so that var name is taken. However, Storybook automatically generates titles for the components based on the story name, thus I had to call it `_LoginForm` so that the story would be titled "Login Form"

Reply:
Blehhhh. I see, but that's stlil bleh lol. What about
```tsx
import LoginForm as Form
```

Reply:
Good call!